orc8r/cloud/go/obsidian/swagger/spec_servicer.go

```diff
@@ -26,37 +26,51 @@ import (
 )
 
 type specServicer struct {
-	spec string
+	partialSpec    string
+	standAloneSpec string
 }
 
 // NewSpecServicerFromFile intializes a spec servicer
 // given a service name.
 func NewSpecServicerFromFile(service string) protos.SwaggerSpecServer {
 	service = strings.ToLower(service)
-	path := getSpecPath(service)
-	data, err := ioutil.ReadFile(path)
+	partialPath, standalonePath := getSpecPaths(service)
+	partial, err := ioutil.ReadFile(partialPath)
 	if err != nil {
-		// Swallowing ReadFile error because the service should
-		// continue to run even if it can't find its Swagger spec file.
+		// Swallowing ReadFile error because the service should continue to
```

Comment:
Nit: same as above, no need to specify the name of the function whose error you're swallowing